app/editor/src/features/admin/ingest-types/IngestTypeList.tsx

```diff
@@ -41,7 +41,7 @@ export const IngestTypeList: React.FC = () => {
       <GridTable
         columns={columns}
         header={IngestTypeFilter}
-        manualPageSize
+        paging={{ pageSizeOptions: { fromLocalStorage: true } }}
```

Comment:
By default local storage is not used, as it would result in sharing the same value everywhere.  I could provide a separate key for each instance, but it isn't a priority.